spec/services/mapquest_service_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+describe MapQuestService do
+  it ".get_coordinates" do
+    service = MapQuestService.new
+    map_info = service.get_coordinates("Anaheim, CA")
+
+    expect(map_info[:results]).to be_an(Array)
+    expect(map_info[:results][0][:locations][0][:latLng][:lat]).to eq(33.834492) #lat
+    expect(map_info[:results][0][:locations][0][:latLng][:lng]).to eq(-117.915638) #long
```

Comment:
The api call is kind of long if I want to get just the lat and long in the map model